document/core/appendix/properties.rst

```diff
@@ -628,16 +628,19 @@ To that end, all previous typing judgements :math:`C \vdash \X{prop}` are genera
 :math:`\DELEGATEadm\{l\}~\instr^\ast~\END`
 ..........................................
 
-.. todo::
-   Add prose.
+* The instruction sequence :math:`\instr^\ast` must be :ref:`valid <valid-instr-seq>` with some type :math:`[]\to[t^\ast]`.
+
+* The label :math:`C.\CLABELS[l+1]` must be defined in the context.
+
+* Then the compound instruction is valid with type :math:`[] \to [t^\ast]`.
 
 .. math::
    \frac{
-     S; C,\CLABELS\,[t^\ast] \vdashinstrseq \instr^\ast : [] \to [t^\ast]
+     S; C \vdashinstrseq \instr^\ast : [] \to [t^\ast]
      \qquad
-     C.\CLABELS[l] = [t_0^\ast]
+     C.\CLABELS[l+1] = [t_0^\ast]
    }{
-     S; C,\CLABELS\,[t^\ast] \vdashadmininstr \DELEGATEadm\{l\}~\instr^\ast~\END : [] \to [t^\ast]
```

Comment:
Can we keep the previous version, as a way to avoid the explicit arithmetics on the label?

Reply:
Done.